pytradfri/smart_task.py

```diff
@@ -177,6 +179,15 @@ def task_control(self) -> TaskControl:
         """Control a task."""
         return TaskControl(self, self.state, self.path, self._gateway)
 
+    @property
+    def delta_time_gateway_local(self) -> timedelta:
+        """Return difference between local time and time set on gateway."""
+        return timedelta(0)
+
+    @delta_time_gateway_local.setter
```

Comment:
Just make it an attribute. I don't think we need a property.

Reply:
I pushed before I saw this comment, will refactor.